css/main.css

```diff
@@ -1,3 +1,222 @@
+Body: Quicksand - Light 300
+https://fonts.google.com/specimen/Quicksand
+
+Colors
+Dark beige: #E6DACD
+Light beige: #F4ECE6
+Royal blue: #0150FD
+Black: #000
+White: #FFF */
 
+/* Add your CSS here */
 /* Dont' forget to link this file to your HTML in the <head> */
+:root {
+  --dark-beige: #e6dacd;
+  --light-beige: #f4ece6;
+  --royal-blue: #0150fd;
+  --black: #000;
+  --white: #fff;
+}
+
+body {
+  width: 320px;
+  margin: 0;
+  padding: 0;
+  font-family: "Quicksand", sans-serif;
+}
+/*Headings: font-family: "Poppins", sans-serif; */
+.content {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-rows: auto;
+  grid-template-areas:
+    "header"
+    "intro"
+    "main"
+    "footer";
+}
+
+/* header style */
+header {
+  padding-left: 6%;
+  padding-top: 4%;
+  margin-bottom: 36%;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-auto-rows: auto;
+  grid-template-areas:
+    "par par par ham"
+    ". . .  ham";
+}
+.hamburger-menu {
+  grid-area: ham;
+  padding-top: 18%;
+  height: 20px;
+  padding-left: 20px;
+}
+
+main {
+  position: relative;
+  padding-top: 10px;
+}
+.profile-img {
+  display: block;
+  border-radius: 50%;
+  width: 44%;
+  margin: 15px auto 18px auto;
+}
+
+header > p:nth-child(1) {
+  margin: 0;
+  display: grid;
+  grid-area: par;
+  font-family: "Poppins", sans-serif;
+  line-height: 1em;
+  font-size: 20px;
+}
+header > p:nth-child(2) {
+  margin: 0;
+  display: grid;
+  grid-area: par;
+  padding-top: 10%;
+  padding-bottom: 10px;
+  line-height: 2em;
+  letter-spacing: 0.02em;
+  font-size: 14px;
+}
+h1 {
+  font-family: "Poppins", sans-serif;
+}
+.hero {
+  position: relative;
+  background-color: rgb(248, 234, 234);
+  margin-left: 6%;
+  margin-right: 2%;
+  padding-bottom: 50px;
+  padding-top: 15px;
+  margin-top: -37%;
+  box-shadow: -5px 5px 15px var(--black);
+}
+.profile-name {
+  width: 22%;
+  border-bottom: 2px solid blue;
+  padding-bottom: 10px;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  font-size: 20px;
+}
+
+.buttons {
+  padding-top: 25px;
+  padding-bottom: 25px;
+  text-align: center;
+  display: flex;
+  justify-content: space-evenly;
+}
+.buttons > button:nth-child(2) {
+  background-color: var(--white);
+  color: var(--black);
+}
+button {
+  width: 8rem;
+  height: 2rem;
+  border-top-left-radius: 50px;
+  border-bottom-left-radius: 50px;
+  border-top-right-radius: 50px;
+  border-bottom-right-radius: 50px;
+  border: var(--black) 1px solid;
+  background-color: var(--royal-blue);
+  color: var(--white);
+  font-weight: bold;
+}
+main .icons {
+  width: 100%;
+  margin-bottom: 10px;
+  background-color: white;
+  justify-content: center;
+  position: absolute;
+  z-index: 10;
+}
+.icons img {
+  padding: 8px;
+}
+/* intro style */
+.intro {
+  background-color: burlywood;
+  position: absolute;
+  z-index: -1;
+  top: 50px;
+  padding: 260px 28px 21px 28px;
+}
+
+/* footer style */
+footer {
```

Comment:
Footer is looking really good, good use of grid here, just remember that the child doesn't need to have display: grid